MediaBrowser.WebDashboard/dashboard-ui/bower_components/emby-webcomponents/emby-radio/emby-radio.css

```diff
@@ -103,8 +103,8 @@
 }
 
 .mdl-radio__button:checked:focus+.mdl-radio__label+.mdl-radio__outer-circle+.mdl-radio__inner-circle {
-    -webkit-box-shadow: 0 0 0 10px rgba(82, 181, 75, .26);
-    box-shadow: 0 0 0 10px rgba(82, 181, 75, .26)
+    -webkit-box-shadow: 0 0 0 10px rgba(0,164,220, .26);
```

Comment:
I'd rather we use hex here, too, so it's easier to search later. Don't know if that is possible in css. If not then using rgb instead of hex everywhere might be the way to go.

Reply:
Hmm, possibly better idea - add a CSS comment next to rgba lines containing hex color, that way all places where theming colors are used will be searchable.

Reply:
I mean, those are RGBA not RGB. In some ways, having RGB and RGB be delineated by dec/hex is nice.

Regardless, if I do get around to the theme system fix up, I plan to mostly redo the existing themes as part of cleaning.